guide/command-handling/adding-features.md

```diff
@@ -386,6 +386,62 @@ No more manually editing your help command! If you aren't completely satisfied w
 If you want to add categories or other information to your commands you can simply add properties reflecting it to your `module.exports`. If you only want to show a subset of commands remember that `commands` is a Collection you can <branch version="11.x" inline>[filter](https://discord.js.org/#/docs/main/11.5.1/class/Collection?scrollTo=filter)</branch><branch version="12.x" inline>[filter](https://discord.js.org/#/docs/collection/master/class/Collection?scrollTo=filter)</branch> to fit your specific needs!
 :::
 
+## Reloading commands
+
+When writing your commands, you may find it tedious to restart bot every time you want to test even slightest change in your code. However, if you have a command handler, reloading commands can be done with a single command.
+
+Create new command file and paste in the usual format:
+
+```js
+module.exports = {
+	name: 'reload',
+	description: 'Reloads a command',
+	execute(message, args) {
+		// ...
+	},
+};
+```
+
+In this command, you will be using command's name or alias as an only argument. First off, you need to check if command you want to reload exists. This can be done in similiar fashion as getting a command in your main file.  
+Note that you can skip first line if you use [argument checker](/command-handling/adding-features.html#required-arguments) from above:
+
+```js
+if (!args.length) return message.channel.send(`You didn't pass any command to reload, ${message.author}!`);
+const commandName = args[0].toLowerCase();
+const command = message.client.commands.get(commandName)
+	|| message.client.commands.find(cmd => cmd.aliases && cmd.aliases.includes(commandName));
+
+if (!command) return message.channel.send(`There is no command with name or alias \`${commandName}\`, ${message.author}!`);
+```
+
+`message.client`? Yes! Every library specific object has `client` as a property. That means that you don't have to pass it as a parameter to commands to access for example `client.guilds` or `client.commands` which you will need now.
```

Comment:
I suggest rephrasing this to a more factual approach and refactoring it into a tip element
"`message.client` here refers to..." or something along those lines